### PR TITLE
add signed speed

### PR DESCRIPTION
**Pull request by yawei-ye**
# Changes
- add signed speed
- [x] add unit test

### CODE REVIEW

File: l5kit/l5kit/sampling/agent_sampling.py [hunk 1]
```diff
@@ -99,6 +99,12 @@ def compute_agent_velocity(
     return history_vels_mps, future_vels_mps
 
 
+def compute_signed_speeds(velocities_mps: np.ndarray) -> np.ndarray:
```
Comment (lucabergamini) [inline l5kit/l5kit/sampling/agent_sampling.py:102]: docstring for this

File: l5kit/l5kit/sampling/agent_sampling.py [hunk 2]
```diff
@@ -275,6 +282,8 @@ def generate_agent_sample(
         "target_positions": future_positions_m,
         "target_yaws": future_yaws_rad,
         "target_velocities": future_vels_mps,
+        "target_speeds": future_speeds_mps,
```
Comment (lucabergamini) [inline l5kit/l5kit/sampling/agent_sampling.py:285]: so this will be the new target for speed right?

**Review by lucabergamini — COMMENTED** [review]
LGTM!, one minor

File: l5kit/l5kit/sampling/agent_sampling.py (hunk 2 above)
Comment (yawei-ye) [reply l5kit/l5kit/sampling/agent_sampling.py:285]: correct

Comment (yawei-ye) [reply l5kit/l5kit/sampling/agent_sampling.py:285]: by the way, it is of shape [num_future_frames] and can be used to train model with longer time horizon.
I created another `"target_speed": future_speeds_mps[:1]` just for not breaking our existing model trained with single speed

Comment (lucabergamini) [reply l5kit/l5kit/sampling/agent_sampling.py:285]: yeah I noticed that, good point

**Review by lucabergamini — APPROVED** [review]

**Pull request merged**